src/room.html

```diff
@@ -24,13 +24,69 @@
 
         <a-assets>
             <a-progressive-asset
-                id="bot-skinned-mesh"
+                id="botdefault"
                 response-type="arraybuffer"
                 src="./assets/avatars/BotDefault_Avatar_Unlit.glb"
                 high-src="./assets/avatars/BotDefault_Avatar.glb"
                 low-src="./assets/avatars/BotDefault_Avatar_Unlit.glb"
             ></a-progressive-asset>
-            <a-asset-item id="bot-dom-mesh" response-type="arraybuffer" src="./assets/avatars/BotDom_Avatar.glb"></a-asset-item>
+            <a-progressive-asset
```

Comment:
Hmm.. This preloads ALL the avatars meaning the scene wont load until we download quite a large chunk of data, much of which we may never use. This is maybe fine for now but we should be just refering to avatars by URL and then we can just download them as needed. We will then probably have to figure out a way to show download progress for other users avatars.

Reply:
this is just history repeating itself :)

seriously though this is def a temporary solution. we should make sure the download time isn't too crazy, and perhaps we should not block loading the app on the avatars.

my mental model is that the environments "lead the way" wrt forcing us to build the tools and systems we will eventually need for avatars. For example the GLTF bundle work was forced into place by the environment needs, and the asset pipeline and image based customization stuff will probably fall out of the construction kit work. The avatars will probably force us to extend this work (for example, introducing LOD properly) but most of the story around how it all fits together will be in place once we have working dynamic custom environments for people.

Reply:
Yeah, I put in the avatars and the avatars.json manifest knowing it would be thrown away, which is fine, they were not the focus of this PR.